pyproject.toml deprecated syntax

### Issue description

Does this mean that the line in `pyproject.toml` is deprecated ? 

Please use a simple string containing a SPDX expression for `project.license`. You can also use `project.license-files`. (Both options available on setuptools>=77.0.0).

 so, in the `pyproject.toml` file under [project] there is a line : 
license = {text = "MIT"}
 if this is it. what should it look like ?


### Version

0.31.1.dev19+gf8040baf

### Backend

Wayland (experimental)

### Config

```python

```

### Logs

```text
pip install --config-setting backend=wayland .
Processing /home/cjoke/dev/qtile
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... error
  error: subprocess-exited-with-error
  
  × pip subprocess to install backend dependencies did not run successfully.
  │ exit code: 1
  ╰─> [184 lines of output]
      Collecting pywlroots<0.18.0,>=0.17.0
        Using cached pywlroots-0.17.0.tar.gz (99 kB)
        Installing build dependencies: started
        Installing build dependencies: finished with status 'done'
        Getting requirements to build wheel: started
        Getting requirements to build wheel: finished with status 'done'
        Installing backend dependencies: started
        Installing backend dependencies: finished with status 'done'
        Preparing metadata (pyproject.toml): started
        Preparing metadata (pyproject.toml): finished with status 'error'
        error: subprocess-exited-with-error
      
        × Preparing metadata (pyproject.toml) did not run successfully.
        │ exit code: 1
        ╰─> [159 lines of output]
            /tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/config/_apply_pyprojecttoml.py:82: SetuptoolsDeprecationWarning: `project.license` as a TOML table is deprecated
            !!
      
                    ********************************************************************************
                    Please use a simple string containing a SPDX expression for `project.license`. You can also use `project.license-files`. (Both options available on setuptools>=77.0.0).
      
                    By 2026-Feb-18, you need to update your project and remove deprecated calls
                    or your builds will no longer be supported.
      
                    See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.
                    ********************************************************************************
      
            !!
              corresp(dist, value, root_dir)
            wlroots/__pycache__/_cffi__x58fcf130x93e86b51.c:269:10: fatal error: wlr/version.h: No such file or directory
              269 | #include <wlr/version.h>
                  |          ^~~~~~~~~~~~~~~
            compilation terminated.
            Traceback (most recent call last):
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/_distutils/spawn.py", line 87, in spawn
                subprocess.check_call(cmd, env=_inject_macos_ver(env))
                ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
              File "/home/cjoke/.pyenv/versions/3.13.2/lib/python3.13/subprocess.py", line 421, in check_call
                raise CalledProcessError(retcode, cmd)
            subprocess.CalledProcessError: Command '['/usr/bin/gcc', '-fno-strict-overflow', '-Wsign-compare', '-DNDEBUG', '-g', '-O3', '-Wall', '-fPIC', '-I/home/cjoke/.pyenv/versions/3.13.2/include/python3.13', '-c', 'wlroots/__pycache__/_cffi__x58fcf130x93e86b51.c', '-o', '/tmp/pip-install-7gcbd_t0/pywlroots_336e7e95f93548fca9df4d1c75baeccd/wlroots/__pycache__/wlroots/__pycache__/_cffi__x58fcf130x93e86b51.o']' returned non-zero exit status 1.
      
            The above exception was the direct cause of the following exception:
      
            Traceback (most recent call last):
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/_distutils/compilers/C/unix.py", line 221, in _compile
                self.spawn(compiler_so + cc_args + [src, '-o', obj] + extra_postargs)
                ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/_distutils/compilers/C/base.py", line 1158, in spawn
                spawn(cmd, dry_run=self.dry_run, **kwargs)
                ~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/_distutils/spawn.py", line 93, in spawn
                raise DistutilsExecError(
                    f"command {_debug(cmd)!r} failed with exit code {err.returncode}"
                ) from err
            distutils.errors.DistutilsExecError: command '/usr/bin/gcc' failed with exit code 1
      
            During handling of the above exception, another exception occurred:
      
            Traceback (most recent call last):
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/cffi/ffiplatform.py", line 48, in _build
                dist.run_command('build_ext')
                ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/dist.py", line 1104, in run_command
                super().run_command(command)
                ~~~~~~~~~~~~~~~~~~~^^^^^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/_distutils/dist.py", line 1021, in run_command
                cmd_obj.run()
                ~~~~~~~~~~~^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/command/build_ext.py", line 99, in run
                _build_ext.run(self)
                ~~~~~~~~~~~~~~^^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/_distutils/command/build_ext.py", line 368, in run
                self.build_extensions()
                ~~~~~~~~~~~~~~~~~~~~~^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/_distutils/command/build_ext.py", line 484, in build_extensions
                self._build_extensions_serial()
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/_distutils/command/build_ext.py", line 510, in _build_extensions_serial
                self.build_extension(ext)
                ~~~~~~~~~~~~~~~~~~~~^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/command/build_ext.py", line 264, in build_extension
                _build_ext.build_extension(self, ext)
                ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/_distutils/command/build_ext.py", line 565, in build_extension
                objects = self.compiler.compile(
                    sources,
                ...<5 lines>...
                    depends=ext.depends,
                )
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/_distutils/compilers/C/base.py", line 655, in compile
                self._compile(obj, src, ext, cc_args, extra_postargs, pp_opts)
                ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/_distutils/compilers/C/unix.py", line 223, in _compile
                raise CompileError(msg)
            distutils.compilers.C.errors.CompileError: command '/usr/bin/gcc' failed with exit code 1
      
            During handling of the above exception, another exception occurred:
      
            Traceback (most recent call last):
              File "wlroots/ffi_build.py", line 36, in load_wlroots_version
                lib = ffi.verify("#include <wlr/version.h>")
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/cffi/api.py", line 468, in verify
                lib = self.verifier.load_library()
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/cffi/verifier.py", line 105, in load_library
                self._compile_module()
                ~~~~~~~~~~~~~~~~~~~~^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/cffi/verifier.py", line 201, in _compile_module
                outputfilename = ffiplatform.compile(tmpdir, self.get_extension())
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/cffi/ffiplatform.py", line 20, in compile
                outputfilename = _build(tmpdir, ext, compiler_verbose, debug)
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/cffi/ffiplatform.py", line 54, in _build
                raise VerificationError('%s: %s' % (e.__class__.__name__, e))
            cffi.VerificationError: CompileError: command '/usr/bin/gcc' failed with exit code 1
      
            During handling of the above exception, another exception occurred:
      
            Traceback (most recent call last):
              File "/home/cjoke/.pyenv/versions/3.13.2/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 389, in <module>
                main()
                ~~~~^^
              File "/home/cjoke/.pyenv/versions/3.13.2/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 373, in main
                json_out["return_val"] = hook(**hook_input["kwargs"])
                                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^
              File "/home/cjoke/.pyenv/versions/3.13.2/lib/python3.13/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 175, in prepare_metadata_for_build_wheel
                return hook(metadata_directory, config_settings)
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 374, in prepare_metadata_for_build_wheel
                self.run_setup()
                ~~~~~~~~~~~~~~^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/build_meta.py", line 317, in run_setup
                exec(code, locals())
                ~~~~^^^^^^^^^^^^^^^^
              File "<string>", line 34, in <module>
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/__init__.py", line 117, in setup
                return distutils.core.setup(**attrs)
                       ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/_distutils/core.py", line 148, in setup
                _setup_distribution = dist = klass(attrs)
                                             ~~~~~^^^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/dist.py", line 323, in __init__
                _Distribution.__init__(self, dist_attrs)
                ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/_distutils/dist.py", line 309, in __init__
                self.finalize_options()
                ~~~~~~~~~~~~~~~~~~~~~^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/dist.py", line 786, in finalize_options
                ep(self)
                ~~^^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/setuptools/dist.py", line 806, in _finalize_setup_keywords
                ep.load()(self, ep.name, value)
                ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/cffi/setuptools_ext.py", line 216, in cffi_modules
                add_cffi_module(dist, cffi_module)
                ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/cffi/setuptools_ext.py", line 49, in add_cffi_module
                execfile(build_file_name, mod_vars)
                ~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^
              File "/tmp/pip-build-env-a962oak3/overlay/lib/python3.13/site-packages/cffi/setuptools_ext.py", line 25, in execfile
                exec(code, glob, glob)
                ~~~~^^^^^^^^^^^^^^^^^^
              File "wlroots/ffi_build.py", line 3132, in <module>
                check_version()
                ~~~~~~~~~~~~~^^
              File "wlroots/ffi_build.py", line 49, in check_version
                wlroots_version = load_wlroots_version()
              File "wlroots/ffi_build.py", line 38, in load_wlroots_version
                lib = importlib.import_module("wlroots").lib
                      ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
              File "/home/cjoke/.pyenv/versions/3.13.2/lib/python3.13/importlib/__init__.py", line 88, in import_module
                return _bootstrap._gcd_import(name[level:], package, level)
                       ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
              File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
              File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
              File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
            ModuleNotFoundError: No module named 'wlroots'
            [end of output]
      
        note: This error originates from a subprocess, and is likely not a problem with pip.
      error: metadata-generation-failed
      
      × Encountered error while generating package metadata.
      ╰─> See above for output.
      
      note: This is an issue with the package mentioned above, not pip.
      hint: See above for details.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× pip subprocess to install backend dependencies did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

### Required

- [x] I have searched past issues to see if this bug has already been reported, and it hasn't been.
- [x] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -13,10 +13,10 @@ build-backend = "builder"
 name = "qtile"
 description = "A pure-Python tiling window manager."
 dynamic = ["version", "readme", "dependencies", "optional-dependencies"]
-license = {text = "MIT"}
+license = "MIT"
+license-files = [ "LICENSE" ]
 classifiers = [
     "Intended Audience :: End Users/Desktop",
-    "License :: OSI Approved :: MIT License",
     "Operating System :: POSIX :: BSD :: FreeBSD",
     "Operating System :: POSIX :: Linux",
     "Programming Language :: Python :: 3 :: Only",

